src/app/context/UserSessionContext.tsx

```diff
@@ -26,7 +26,7 @@ interface IUserSessionContext {
   onSignOut: () => Promise<void>;
 }
 
-const UserSessionContext = React.createContext<IUserSessionContext>(
+export const UserSessionContext = React.createContext<IUserSessionContext>(
```

Comment:
We can omit this export, because we actually have the hook useUserSession at the bottom of this file. You can replace it in Questionform.tsx

Reply:
oh.. I'm not quite sure how to to use it tbh